packages/components/src/Filter/_docs/Filter.stickersheet.stories.tsx

```diff
@@ -10,6 +10,17 @@ const IS_CHROMATIC = isChromatic()
 export default {
   title: "Components/Filter Base",
   parameters: {
+    a11y: {
+      config: {
+        rules: [
+          {
```

Comment:
Does this also need the `globalA11yRules`?

Reply:
Yep